.eslintrc.js

```diff
@@ -1,2 +1,5 @@
-// @generated by expo-module-scripts
-module.exports = require('expo-module-scripts/eslintrc.base.js');
+module.exports = {
+  root: true,
+  extends: ['universe/native', 'universe/web'],
+  ignorePatterns: ['build'],
+};
```

Comment:
Why do we need a custom config?

Reply:
Turns out we didnt! I deleted the file and the next time i ran `yarn` it was recreated from the template. linting still works 🥳